distribution_strategy/parameter_server_training/README.md

```diff
@@ -0,0 +1,173 @@
+# Parameter Server Training Using Distribution Strategies
+
+This directory provides an example of running parameter server training with
+Distribution Strategies.
+
+Please first read the
+[documentation](https://www.tensorflow.org/tutorials/distribute/parameter_server_training)
+of Distribution Strategy for parameter server training. We also assume that readers
+of this page  are familiar with [Google Cloud](https://cloud.google.com/) and
+its [Kubernetes Engine](https://cloud.google.com/kubernetes-engine/).
+
+This directory contains the following files:
+
+-   kubernetes/template.yaml.jinja: jinja template used for generating Kubernetes manifests
+-   kubernetes/render_template.py: script for rendering the jinja template
+-   Dockerfile.resnet_cifar_ps_strategy: a docker file to build the model image
+-   resnet_cifar_ps_strategy.py: script for running any type of parameter server training task based on `TF_CONFIG` environment variable
```

Comment:
"any type of ..." seems too general, maybe just say "a ResNet example using Cifar dataset for parameter server training"

Reply:
Done.